documentation/rfc/distributed.md

```diff
@@ -0,0 +1,36 @@
+Scenario Distributed
+====================
+
+
+This scenario allow us to call multiple engines for computing fallbacks. These engines can be:
+
+  - kraken
+  - geovelo
+  - here
+  - valhalla
+
+The orchestration is a bit complex, as such this documentation will try to explain what is done, and it what order.
```

Comment:
**So** this documentation will try to explain what is done, and **in** what order // or //will try to explain the sequence of what is done